azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participant/grid/cell/ParticipantGridCellAvatarView.kt

```diff
@@ -100,6 +110,23 @@ internal class ParticipantGridCellAvatarView(
         }
     }
 
+    private fun setTextViewDisplayName(displayName: String) {
```

Comment:
All this logic should be in the viewModel. `displayName:` should already be `azure_communication_ui_calling_call_view_calling` or other value.